components/admin/requests/forms/new-applications/GuardianInformationForm.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Text,
+  Stack,
+  FormHelperText,
+  Box,
+  Divider,
+  Button,
+} from '@chakra-ui/react'; // Chakra UI
+import { AttachmentIcon } from '@chakra-ui/icons';
+import { GuardianInformation } from '@tools/components/admin/requests/forms/types'; // Guardian Information Type
+
+type GuardianInformationFormProps = {
+  readonly guardianInformation: GuardianInformation;
+  readonly onChange: (updatedData: GuardianInformation) => void;
+  readonly onFileUpload: (args: Record<string, any>) => void;
+};
+
+/**
+ * GuardianInformationForm Component for allowing users to edit guardian information.
+ *
+ * @param {GuardianInformation} permitHolderInformation Data Structure that holds all guardian information for a client request.
+ * @param onChange Function that uses the updated values from form.
+ */
+export default function GuardianInformationForm({
+  guardianInformation,
+  onChange,
+  onFileUpload,
+}: GuardianInformationFormProps) {
+  return (
+    <>
+      {/* Personal Information Section */}
+      <Box paddingBottom="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {'Guardian/POA Information'}
+        </Text>
+        <Stack direction="row" spacing="20px">
+          <FormControl isRequired>
+            <FormLabel>{'First name'}</FormLabel>
+            <Input
+              value={guardianInformation.firstName}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  firstName: event.target.value,
+                })
```

Comment:
Something I realized was that instead of destructuring the object like this in every `Input` field, we can make a function that outputs and event handler like so:
```typescript
const handleChange = (field: keyof GuardianInformation) => (event: SyntheticEvent) => {
  onChange({
     ...guardianInformation,
    [field]: event.target.value,
  })
}
```

Just to make the code a little cleaner within the JSX

Reply:
I got this error when using this event handler 
`Property 'value' does not exist on type 'EventTarget'`

Reply:
I did this instead and the error was gone. 
`const handleChange = (field: keyof GuardianInformation) => (event: SyntheticEvent) => {
    const target = event.target as HTMLTextAreaElement
    onChange({
      ...guardianInformation,
      [field]: target.value,
    });
  };`

Is this okay?

Reply:
Resolved offline

Reply:
Solution:
```typescript
const handleChange = (field: keyof GuardianInformation): ChangeEventHandler<HTMLInputElement> => (event) => {
    onChange({
       ...guardianInformation,
      [field]: event.target.value,
    })
  };
```
cc @chrischan325